package.json

```diff
@@ -66,7 +66,7 @@
     "portfinder": "^1.0.2"
   },
   "scripts": {
-    "postinstall": "if $BUILD_POSTINSTALL; then ./node_modules/.bin/gulp build; fi",
+    "postinstall": "if [ $BUILD_POSTINSTALL ]; then ./node_modules/.bin/gulp build; fi",
```

Comment:
The old condition was always true, the new condition works correctly in the shell but not as a npm script :astonished: